nucypher/characters/control/interfaces.py

```diff
@@ -192,18 +192,14 @@ def retrieve(self,
 
         self.implementer.join_policy(label=label, publisher_verifying_key=alice_verifying_key)
 
-        if self.implementer.federated_only:
-            from nucypher.policy.maps import TreasureMap as _MapClass
-        else:
-            from nucypher.policy.maps import SignedTreasureMap as _MapClass
+        from nucypher.policy.maps import EncryptedTreasureMap
```

Comment:
Such a relief to see this conditional importing cleaned up.  Can this line be moved to the top of the module without a circular dependency?